proto/directions.proto

```diff
@@ -142,7 +142,7 @@ message DirectionsLeg {
     optional string text_instruction = 2;                    // text instruction
     repeated StreetName street_name = 3;                     // street names
     optional float length = 4;                               // kilometers or miles based on units
```

Comment:
should `length` be switched to `double` too?

Reply:
i thought about it but it seemed somewhat unrelated to the PR at hand. i could expand the pr to test lengths as well. is that the concensus?

Reply:
I think length can be handled in a separate PR. I was just a bit confused as I saw spots in the code where you change the precision of `distance` in the response.

Reply:
i was thinking to be consistent since `distance` was returned as double

Reply:
i changed the decimal places in json but not the internal precision (which is float). we should change internal distance to be double but at the moment that is a larger change